Gemfile.lock

```diff
@@ -0,0 +1,92 @@
+GEM
```

Comment:
This seems like a lot of...stuff. I can't quite tell what the releasatron does that warrants all of it (vs say a bit of sed).